packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -641,3 +642,18 @@ function getNumberOfPages(items: QuestionnaireItem[]): number {
   });
   return pages.length > 0 ? items.length : 0;
 }
+
+function addTargetTypes(item: QuestionnaireItem): string[] {
+  if (item.type !== QuestionnaireItemType.reference) {
+    return [];
+  }
+  const extensions = item.extension?.filter(
+    (e) => e.url === 'http://hl7.org/fhir/R4/extension-questionnaire-referenceresource.html'
```

Comment:
Based on [this spec](http://hl7.org/fhir/R4/extension-questionnaire-referenceresource.html:  

> URL for this extension:
> 
> http://hl7.org/fhir/StructureDefinition/questionnaire-referenceResource

Reply:
Value should be valueCode